### PR TITLE
Added example on how to adjust the validation message for specific form field

### PR DESCRIPTION
**Pull request by vasilii-b**
## Purpose of this pull request

This pull request (PR) add example on how to change validation message for form fields separately using existing validation rules.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/validations/custom-validation.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3.4/app/code/Magento/CatalogSearch/view/frontend/templates/advanced/form.phtml#L161-L162

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md [hunk 1]
```diff
@@ -50,3 +50,34 @@ define(['jquery'], function($) {
   }
 });
 ```
+
+Also, it is possible to adjust existing error message for some specific form field separately.
+
+This is implemented in the core codebase in scope of the [`Magento_CatlogSearch` module]({{ site.mage2bloburl }}/{{page.guide_version}}/app/code/Magento/CatalogSearch/view/frontend/templates/advanced/form.phtml).
+
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md]: Hi @vasilii-b. Thank you for the great example. Could I ask you to add an extended explanation of what exactly needs to be done in order to adjust the message for a specific field? 

The example below shows how it's done in the core but it does not explain the process explicitly. The question that may appear, do we need to modify the `errorPlacement` for adjusting the message or it's enough to add the corresponding item into the `messages` object? 

Thank you!

Comment (vasilii-b) [reply src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md]: Hi @rogyar,
Thanks for looking into it.
Answering your question - no, the `errorPlacement` does not affect the modification of the message.
The `errorPlacement` adjusts where the validation message will show up when the validation rule not passed.

I'll update the file accordingly.
Thank you!

**Comment by vasilii-b** [comment]
Hi @rogyar,
An extended explanation has been added.
Thank you!

File: src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md (hunk 1 above)
Comment (rogyar) [reply src/guides/v2.3/frontend-dev-guide/validations/custom-validation.md]: Awesome, thank you!

**Comment by dobooth** [comment]
running tests

**Comment by hguthrie** [comment]
@dobooth  The hand of fate has been dealt. You must add a section for `whatsnew` or change your labeling strategy.

**Pull request merged**